include/crm/stonith-ng.h

```diff
@@ -75,7 +76,9 @@ enum op_state
 enum stonith_namespace {
     st_namespace_invalid,
     st_namespace_any,
-    st_namespace_internal,  // Implemented internally by Pacemaker
+    st_namespace_unused,  // Previously implemented
+                          // internally by Pacemaker
+                          // can we toss it?
```

Comment:
Public API, we should leave it alone. Doesn't hurt to keep around for potential future use

Reply:
Could warn if used instead ...

Reply:
I don't think it's a problem. The only way external code can use this is to e.g. list all devices in a given namespace. If there are no internal devices, it returns an empty list. Not really any different from asking for a list of lha fence agents when none are installed locally.

Reply:
Don't think it is a problem either. Just thought it would be nice to have at least a warning for something that is probably a mistake. But of course we can just go back to the previous definition and just state in the header that it isn't used atm.

Reply:
reverted change as we are still using it now for rolling upgrade support.